<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme" content="hugo-academic">
  <meta name="generator" content="Hugo 0.43" />
  <meta name="author" content="Computational Systems Medicine">

  
  
  
  
  <meta name="description" content="">

  
  <link rel="alternate" hreflang="en-us" href="/publication/">

  


  

  
  
  <meta name="theme-color" content="#0095eb">
  
  
  
  
    
  
  
  
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">
  
  
  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Montserrat:400,700%7cRoboto:400,400italic,700%7cRoboto&#43;Mono">
  
  <link rel="stylesheet" href="/styles.css">
  

  

  
  <link rel="alternate" href="/publication/index.xml" type="application/rss+xml" title="The Vera-Licona Lab">
  <link rel="feed" href="/publication/index.xml" type="application/rss+xml" title="The Vera-Licona Lab">
  

  <link rel="manifest" href="/site.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="/publication/">

  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="twitter:site" content="@CompSysMed">
  <meta property="twitter:creator" content="@CompSysMed">
  
  <meta property="og:site_name" content="The Vera-Licona Lab">
  <meta property="og:url" content="/publication/">
  <meta property="og:title" content="Publications | The Vera-Licona Lab">
  <meta property="og:description" content="">
  <meta property="og:locale" content="en-us">
  
  <meta property="og:updated_time" content="2017-01-01T00:00:00-05:00">
  

  
  

  <title>Publications | The Vera-Licona Lab</title>

</head>
<body id="top" data-spy="scroll" data-target="#toc" data-offset="71" >

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      
      <a class="navbar-brand" href="/"><img src="/img/uconnhealth_wordmark_stacked_blue.jpg" alt="The Vera-Licona Lab"></a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      
      <ul class="nav navbar-nav navbar-right">
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#about">
            
            <span>Home</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#projects">
            
            <span>Projects</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#team">
            
            <span>Team</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#publications">
            
            <span>Publications</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#tools">
            
            <span>Tools</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#outreach">
            
            <span>Education&amp;Outreach</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#contact">
            
            <span>Contact</span>
            
          </a>
        </li>

        
        
      

      
      </ul>

    </div>
  </div>
</nav>





<div class="container">
  <div class="row">
    <div class="col-md-12">
      <h1>Publications</h1>

      

      
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      

      <p>
        Filter by type:

        <select class="pub-filters pubtype-select" data-filter-group="pubtype">
          <option value="*">All</option>
          
          <option value=".pubtype-1">
            
          </option>
          
          <option value=".pubtype-2">
            
          </option>
          
          <option value=".pubtype-6">
            
          </option>
          
        </select>

        <select class="pub-filters" data-filter-group="year">
          <option value="*">All</option>
          
          
          
          <option value=".year-2019">
            2019
          </option>
          
          <option value=".year-2018">
            2018
          </option>
          
          <option value=".year-2017">
            2017
          </option>
          
          <option value=".year-2016">
            2016
          </option>
          
          <option value=".year-2014">
            2014
          </option>
          
          <option value=".year-2013">
            2013
          </option>
          
          
        </select>
      </p>

      <div id="container-publications">
        

        
          
        

        <div class="grid-sizer col-md-12 isotope-item pubtype-2 year-2019">
          
            <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
  

  <h3 class="article-title" itemprop="name">
    <a href="/publication/myd88/" itemprop="url">Macrophage mediated recognition and clearance of Borrelia burgdorferinelicits MyD88-dependent and -independent phagosomal signals that contribute to phagocytosis and inflammation</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    Macrophages play prominent roles in bacteria recognition and clearance, including Borrelia burgdorferi (Bb), the Lyme disease spirochete. To elucidate mechanisms by which MyD88/TLR signaling enhances clearance of Bb by macrophages, we studied Bb-infected wildtype (WT) and MyD88−/− mice and Bb-stimulated bone marrow-derived macrophages (BMDMs). Bb-infected MyD88−/− mice show increased bacterial burdens, macrophage infiltration and altered gene expression in inflamed heart tissue. MyD88−/− BMDMs exhibit impaired uptake of spirochetes but comparable maturation of phagosomes following internalization of spirochetes. RNA-sequencing of infected WT and MyD88−/− BMDMs identified a large cohort of differentially expressed MyD88-dependent genes involved in re-organization of actin and cytoskeleton during phagocytosis along with several MyD88-independent chemokines involved in inflammatory cell recruitment. We computationally generated networks which identified several MyD88-independent master regulators (Cxcl2 and Vcam1) and MyD88-dependent intermediate proteins (Rhoq and Cyfip1) that are known to mediate inflammation and phagocytosis respectively. These results provide mechanistic insights into MyD88-mediated phagosomal signaling enhancing Bb uptake and clearance.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Sarah J. Benjamin, Kelly L. Hawley, <strong>Paola Vera-Licona</strong> &dagger;, Carson J. La Vake, Jorge L. Cervantes, Rachel Burns, Oscar Luo, Yijun Ruan, Melissa J. Caimano, Justin D. Radolf, Juan C. Salazar
    
  </div>

  <div class="pub-publication">
    
    <a href="https://www.biorxiv.org/content/10.1101/593566v1" target="_blank">https://www.biorxiv.org/content/10.1101/593566v1</a>,
    
    2019
  </div>

  <div class="pub-links">
    



<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/10.1101/593566v1" target="_blank" rel="noopener">
  Preprint
</a>









  
  <a class="btn btn-primary btn-outline btn-xs" href="/project/lymedisease/">
    Project
  </a>
  








  </div>

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-md-12 isotope-item pubtype-1 year-2019">
          
            <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
  

  <h3 class="article-title" itemprop="name">
    <a href="/publication/discretization/" itemprop="url">Benchmarking Time-Series Data Discretization on Inference Methods</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    The rapid development in quantitatively measuring DNA, RNA, and protein has stimulated a great  interest  in the  development  of  reverse-engineering  methods,  that  is,  data-driven  approaches to infer the network structure or a dynamical model of the system. Many reverse engineering methods require  discrete  quantitative  data  as  input,  while  many  experimental  data  are  continuous.  While  data discretization has been considered as a pre-processing step, some studies on the reverse-engineering of intracellular networks have started to reveal the impact that the choice of data discretization has on the performance of reverse-engineering methods. However, more comprehensive studies are still greatly needed to systematically and quantitatively understand the impact that discretization methods have on inference methods. Furthermore, there is an urgent need for systematic comparative methods that can help select between discretization methods. In  this  work,  we  consider  4  different  published  intracellular  networks  with  their  respective time-series datasets.  We consider each publication’s original reverse-engineering method and original datasets,  and  discretized  the  data  with  different  discretization  methods.  Across  the  4  studied  cases, changing the data discretization to a more appropriate one, improved the reverse engineering methods’ performance  by  either  increasing  the  number  of  inferred  true  positives  or  decreasing  the  number  of inferred false positives. We observed no universal best discretization method across different time-series data sets.  Thus,  we propose TEDIE, a two-step evaluation metric for ranking discretization methods for time-series data. The underlying assumption of TEDIE is that an optimal discretization method should preserve the dynamic patterns observed in the original data across all variables. We used the 4 different published data sets and networks to validate TEDIE.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Yuezhe Li, Tiffany Jann, <strong>Paola Vera-Licona</strong> &dagger;
    
  </div>

  <div class="pub-publication">
    
    Bioinformatics. doi: 10.1093/bioinformatics/btz036. PMID: 30657860,
    
    2019
  </div>

  <div class="pub-links">
    



<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/early/2018/08/01/378620" target="_blank" rel="noopener">
  Preprint
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.ncbi.nlm.nih.gov/pubmed/30657860" target="_blank" rel="noopener">
  PDF
</a>





<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/VeraLiconaResearchGroup/Benchmarking_TSDiscretizations" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-primary btn-outline btn-xs" href="/project/reverseengineering/">
    Project
  </a>
  








  </div>

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-md-12 isotope-item pubtype-2 year-2018">
          
            <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
  

  <h3 class="article-title" itemprop="name">
    <a href="/publication/signalingflow/" itemprop="url">Computing Signal Transduction in signaling networks modeled as Boolean Networks, Petri Nets and hypergraphs</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    Mathematical frameworks circumventing the need of mechanistic detail exist to build models of signal transduction networks: graphs, hypergraphs, Boolean Networks, and Petri Nets. Predicting how a signal transduces in a signaling network is essential to understand cellular functions and disease. Different formalisms exist to describe how a signal transduces in a given intracellular signaling network represented in the aforementioned modeling frameworks: elementary signaling modes, T-invariants, extreme pathway analysis, elementary flux modes and simple paths. While these signal transduction formalisms are broadly used in their respective frameworks, few studies have been done emphasizing how these signal transduction methodologies compare or relate to each other. We present an overview of how signal transduction networks have been modelled using graphs, hypergraphs, Boolean Networks, and Petri Nets in the literature. We provide a literary review of the different formalisms for capturing signal transduction in a given model of an intracellular signaling network. We also discuss the existing translations between the different modeling frameworks, and the relationships between their corresponding signal transduction representations that have been described in the literature. Furthermore, as a new formalism of signal transduction, we show how minimal functional routes proposed for signaling networks modeled as Boolean Networks can be captured by computing topological factories, a methodology found in the metabolic networks literature. We further show that in the case of an acyclic B-hypergraph, the definitions are equivalent. In directed graphs, it has been shown that computations of elementary modes via its incidence matrix correspond to computations of simple paths and feedback loops. We show that computing elementary modes based on the incidence matrix of a B-hypergraph fails to capture minimal functional routes.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Luis Sordo Vieira, <strong>Paola Vera-Licona</strong> &dagger;
    
  </div>

  <div class="pub-publication">
    
    bioRxiv 272344 <a href="https://doi.org/10.1101/272344" target="_blank">https://doi.org/10.1101/272344</a>,
    
    2018
  </div>

  <div class="pub-links">
    



<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/early/2018/07/10/272344" target="_blank" rel="noopener">
  Preprint
</a>









  
  <a class="btn btn-primary btn-outline btn-xs" href="/project/structurecontrol/">
    Project
  </a>
  








  </div>

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-md-12 isotope-item pubtype-1 year-2017">
          
            <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
  

  <h3 class="article-title" itemprop="name">
    <a href="/publication/quin2/" itemprop="url">Chromatin interaction networks revealed unique connectivity patterns of broad H3K4me3 domains and super enhancers in 3D chromatin</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    Broad domain promoters and super enhancers are regulatory elements that govern cell-specific functions and harbor disease-associated sequence variants. These elements are characterized by distinct epigenomic profiles, such as expanded deposition of histone marks H3K27ac for super enhancers and H3K4me3 for broad domains, however little is known about how they interact with each other and the rest of the genome in three-dimensional chromatin space. Using network theory methods, we studied chromatin interactions between broad domains and super enhancers in three ENCODE cell lines (K562, MCF7, GM12878) obtained via ChIA-PET, Hi-C, and Hi-CHIP assays. In these networks, broad domains and super enhancers interact more frequently with each other compared to their typical counterparts. Network measures and graphlets revealed distinct connectivity patterns associated with these regulatory elements that are robust across cell types and alternative assays. Machine learning models showed that these connectivity patterns could effectively discriminate broad domains from typical promoters and super enhancers from typical enhancers. Finally, targets of broad domains in these networks were enriched in disease-causing SNPs of cognate cell types. Taken together these results suggest a robust and unique organization of the chromatin around broad domains and super enhancers: loci critical for pathologies and cell-specific functions.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Asa Thibodeau, Eladio J. Márquez, Dong-Guk Shin, <strong>Paola Vera-Licona</strong> &dagger;, Duygu Ucar &dagger;
    
  </div>

  <div class="pub-publication">
    
    Scientific Reports. Volume 7, Article number: 14466,
    
    2017
  </div>

  <div class="pub-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://www.nature.com/articles/s41598-017-14389-7" target="_blank" rel="noopener">
  PDF
</a>





<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/UcarLab/CellSpecificNetworks" target="_blank" rel="noopener">
  Code
</a>











  </div>

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-md-12 isotope-item pubtype-1 year-2017">
          
            <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
  

  <h3 class="article-title" itemprop="name">
    <a href="/publication/spleenmacs/" itemprop="url">CD169&#43; marginal zone macrophages orchestrate innate immune responses to bacterial infection</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    Broad domain promoters and super enhancers are regulatory elements that govern cell-specific functions and harbor disease-assoThe spleen is an important site for generating protective immune responses against pathogens. After infection, immune cells undergo rapid reorganization to initiate and maintain localized inflammatory responses; however, the mechanisms governing this spatial and temporal cellular reorganization remain unclear. We show that the strategic position of splenic marginal zone CD169+ macrophages is vital for rapid initiation of antibacterial responses. In addition to controlling initial bacterial growth, CD169+ macrophages orchestrate a second phase of innate protection by mediating the transport of bacteria to splenic T cell zones. This compartmentalization of bacteria within the spleen was essential for driving the reorganization of innate immune cells into hierarchical clusters and for local interferon-γ production near sites of bacterial replication foci. Our results show that both phases of the antimicrobial innate immune response were dependent on CD169+ macrophages, and, in their absence, the series of events needed for pathogen clearance and subsequent survival of the host was disrupted. Our study provides insight into how lymphoid organ structure and function are related at a fundamental level..
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Oriana A. Perez, Stephen T. Yeung, <strong>Paola Vera-Licona</strong>, Pablo A. Romagnoli, Tasleem Samji, Basak B. Ural, Leigh Maher, Masato Tanaka, Kamal M. Khanna &dagger;
    
  </div>

  <div class="pub-publication">
    
    Science Immunology. Vol. 2, Issue 16 (article featured on the cover),
    
    2017
  </div>

  <div class="pub-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="http://immunology.sciencemag.org/content/2/16/eaah5520.long" target="_blank" rel="noopener">
  PDF
</a>








  
  <a class="btn btn-primary btn-outline btn-xs" href="/project/khannamacs/">
    Project
  </a>
  








  </div>

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-md-12 isotope-item pubtype-1 year-2017">
          
            <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
  

  <h3 class="article-title" itemprop="name">
    <a href="/publication/mhss/" itemprop="url">The Minimal Hitting Set Generation Problem: Algorithms and Computation</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    Finding inclusion-minimal hitting sets (MHSs) for a given family of sets is a fundamental combinatorial problem with applications in domains as diverse as Boolean algebra, computational biology, and data mining. Although many algorithms are available in the literature to generate these MHSs, application papers typically consider only a few before selecting one (or introducing a novel algorithm), suggesting the need for a comprehensive survey and performance comparison. We introduce several of these applications, discussing how MHS generation is applied in each domain and which algorithms have been used, providing a unified view of these applications for researchers from diverse areas. We survey twenty-one algorithms for MHS generation from across a variety of domains, considering their history, classification, and useful features. We provide the results of a comprehensive suite of benchmarks of public software implementations of seventeen of these algorithms, including six we implemented ourselves in C++, emphasizing problem instances taken from real applications in the literature. We find that the fastest algorithms in practice are not those with the tightest complexity bounds or those most commonly used in applications, suggesting that, for a given application, benchmarking from across the broad span of available algorithms will enable a better choice. Finally, we provide a public repository of these software implementations as ready-to-use, platform-agnostic Docker containers based on the AlgoRun framework, so interested computational scientists can easily perform similar tests with inputs from their own research areas, either on their own computers or through a convenient Web interface or deploy the algorithms in their own analysis pipelines.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Andrew Gainer-Dewar, <strong>Paola Vera-Licona</strong> &dagger;
    
  </div>

  <div class="pub-publication">
    
    SIAM Journal on Discrete Mathematics. 2017; 31(1):63–100,
    
    2017
  </div>

  <div class="pub-links">
    



<a class="btn btn-primary btn-outline btn-xs" href="https://arxiv.org/abs/1601.02939" target="_blank" rel="noopener">
  Preprint
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://epubs.siam.org/doi/10.1137/15M1055024" target="_blank" rel="noopener">
  PDF
</a>





<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/VeraLiconaResearchGroup/MHSGenerationAlgorithms" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-primary btn-outline btn-xs" href="/project/structurecontrol/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://algorun.org/browse?q=mhs&amp;hPP=100&amp;idx=dev_AlgoRun&amp;p=0&amp;is_v=1" target="_blank" rel="noopener">
  AlgoRunContainer
</a>


  </div>

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-md-12 isotope-item pubtype-1 year-2016">
          
            <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
  

  <h3 class="article-title" itemprop="name">
    <a href="/publication/algorun/" itemprop="url">AlgoRun: a Docker-based packaging system for platform-agnostic implemented algorithms</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    <strong>Motivation</strong>: There is a growing need in bioinformatics for easy-to-use software implementations of algorithms that are usable across platforms. At the same time, reproducibility of computational results is critical and often a challenge due to source code changes over time and dependencies. <strong>Results</strong>: The approach introduced in this paper addresses both of these needs with AlgoRun, a dedicated packaging system for implemented algorithms, using Docker technology. Implemented algorithms, packaged with AlgoRun, can be executed through a user-friendly interface directly from a web browser or via a standardized RESTful web API to allow easy integration into more complex workflows. The packaged algorithm includes the entire software execution environment, thereby eliminating the common problem of software dependencies and the irreproducibility of computations over time. AlgoRun-packaged algorithms can be published on <a href="http://algorun.org" target="_blank">http://algorun.org</a>, a centralized searchable directory to find existing AlgoRun-packaged algorithms.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Abdelrahman Hosny <strong>*</strong>, <strong>Paola Vera-Licona</strong> <strong>*</strong>, Reinhard Laubenbacher &dagger;, Thibauld Favre
    
  </div>

  <div class="pub-publication">
    
    Bioinformatics, Volume 32, Issue 15, 1 August 2016, Pages 2396–2398,
    
    2016
  </div>

  <div class="pub-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://academic.oup.com/bioinformatics/article/32/15/2396/1742878" target="_blank" rel="noopener">
  PDF
</a>





<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/algorun" target="_blank" rel="noopener">
  Code
</a>










<a class="btn btn-primary btn-outline btn-xs" href="http://algorun.org" target="_blank" rel="noopener">
  AlgoRunWebsite
</a>


  </div>

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-md-12 isotope-item pubtype-1 year-2016">
          
            <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
  

  <h3 class="article-title" itemprop="name">
    <a href="/publication/quin1/" itemprop="url">QuIN: A Web Server for Querying and Visualizing Chromatin Interaction Networks</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    Recent studies of the human genome have indicated that regulatory elements (e.g. promoters and enhancers) at distal genomic locations can interact with each other via chromatin folding and affect gene expression levels. Genomic technologies for mapping interactions between DNA regions, e.g., ChIA-PET and HiC, can generate genome-wide maps of interactions between regulatory elements. These interaction datasets are important resources to infer distal gene targets of non-coding regulatory elements and to facilitate prioritization of critical loci for important cellular functions. With the increasing diversity and complexity of genomic information and public ontologies, making sense of these datasets demands integrative and easy-to-use software tools. Moreover, network representation of chromatin interaction maps enables effective data visualization, integration, and mining. Currently, there is no software that can take full advantage of network theory approaches for the analysis of chromatin interaction datasets. To fill this gap, we developed a web-based application, QuIN, which enables: 1) building and visualizing chromatin interaction networks, 2) annotating networks with user-provided private and publicly available functional genomics and interaction datasets, 3) querying network components based on gene name or chromosome location, and 4) utilizing network based measures to identify and prioritize critical regulatory targets and their direct and indirect interactions.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Asa Thibodeau, Eladio J. Márquez, Oscar Luo, Yijun Ruan, Francesca Menghi, Dong-Guk Shin, Michael L. Stitzel, <strong>Paola Vera-Licona</strong>, Duygu Ucar &dagger;
    
  </div>

  <div class="pub-publication">
    
    PLoS Comput Biol. 2016 Jun 23;12(6):e1004809. PubMed PMID: 27336171; PubMed Central PMCID: PMC4919057,
    
    2016
  </div>

  <div class="pub-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004809" target="_blank" rel="noopener">
  PDF
</a>





<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/UcarLab/QuIN/" target="_blank" rel="noopener">
  Code
</a>










<a class="btn btn-primary btn-outline btn-xs" href="http://quin.jax.org" target="_blank" rel="noopener">
  QuinWebServer
</a>


  </div>

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-md-12 isotope-item pubtype-1 year-2014">
          
            <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
  

  <h3 class="article-title" itemprop="name">
    <a href="/publication/react/" itemprop="url">An algebra-based method for inferring gene regulatory networks</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    <strong>Motivation</strong>: There is a growing need in bioinformatics for easy-to-use software implementations of algorithms that are usable across platforms. At the same time, reproducibility of computational results is critical and often a challenge due to source code changes over time and dependencies. <strong>Results</strong>: The approach introduced in this paper addresses both of these needs with AlgoRun, a dedicated packaging system for implemented algorithms, using Docker technology. Implemented algorithms, packaged with AlgoRun, can be executed through a user-friendly interface directly from a web browser or via a standardized RESTful web API to allow easy integration into more complex workflows. The packaged algorithm includes the entire software execution environment, thereby eliminating the common problem of software dependencies and the irreproducibility of computations over time. AlgoRun-packaged algorithms can be published on <a href="http://algorun.org" target="_blank">http://algorun.org</a>, a centralized searchable directory to find existing AlgoRun-packaged algorithms.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    <strong>Paola Vera-Licona</strong> &dagger;, Abdul Jarrah, Luis David Garcia-Puente, John McGee, Reinhard Laubenbacher
    
  </div>

  <div class="pub-publication">
    
    BMC Syst Biol. 2014 Mar 26;8:37. PubMed PMID: 24669835 (chosen as Editor’s Picks),
    
    2014
  </div>

  <div class="pub-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-8-37" target="_blank" rel="noopener">
  PDF
</a>





<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/veralicona/REACT" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-primary btn-outline btn-xs" href="/project/reverseengineering/">
    Project
  </a>
  







<a class="btn btn-primary btn-outline btn-xs" href="http://algorun.org/browse?q=react&amp;hPP=100&amp;idx=dev_AlgoRun&amp;p=0&amp;is_v=1" target="_blank" rel="noopener">
  AlgoRunContainer
</a>


  </div>

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-md-12 isotope-item pubtype-1 year-2013">
          
            <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
  

  <h3 class="article-title" itemprop="name">
    <a href="/publication/ocsana/" itemprop="url">OCSANA: optimal combinations of interventions from network analysis</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    Targeted therapies interfering with specifically one protein activity are promising strategies in the treatment of diseases like cancer. However, accumulated empirical experience has shown that targeting multiple proteins in signaling networks involved in the disease is often necessary. Thus, one important problem in biomedical research is the design and prioritization of optimal combinations of interventions to repress a pathological behavior, while minimizing side-effects. OCSANA (optimal combinations of interventions from network analysis) is a new software designed to identify and prioritize optimal and minimal combinations of interventions to disrupt the paths between source nodes and target nodes. When specified by the user, OCSANA seeks to additionally minimize the side effects that a combination of interventions can cause on specified off-target nodes. With the crucial ability to cope with very large networks, OCSANA includes an exact solution and a novel selective enumeration approach for the combinatorial interventions’ problem.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    <strong>Paola Vera-Licona</strong> &dagger;, Eric Bonnet, Emmanuel Barillot, Andrei Zinovyev
    
  </div>

  <div class="pub-publication">
    
    Bioinformatics, Volume 29, Issue 12, 15 June 2013, Pages 1571–1573,
    
    2013
  </div>

  <div class="pub-links">
    




<a class="btn btn-primary btn-outline btn-xs" href="https://academic.oup.com/bioinformatics/article/29/12/1571/293094" target="_blank" rel="noopener">
  PDF
</a>





<a class="btn btn-primary btn-outline btn-xs" href="http://bioinfo-out.curie.fr/projects/ocsana/OCSANA.html" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-primary btn-outline btn-xs" href="/project/structurecontrol/">
    Project
  </a>
  

  
  <a class="btn btn-primary btn-outline btn-xs" href="/project/cancerreversion/">
    Project
  </a>
  








  </div>

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-md-12 isotope-item pubtype-6 year-2013">
          
            <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
  

  <h3 class="article-title" itemprop="name">
    <a href="/publication/modelgenes1/" itemprop="url">Computational systems biology: Discrete models of gene regulation networks</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    This article describes a 2-3 day workshop offered at regional undergraduate teaching institutions and high schools. Its goal is to use discrete dynamic models, in particular Boolean networks, to illustrate mathematical modeling of biological networks, such as gene regulatory networks, to a broad audience that can include undergraduate faculty, undergraduate students, high school teachers, and even high school students. The workshop covers the basics of biology, mathematical modeling, and model analysis, using the well-known lac operon network in E coli as a model system. The workshop materials can be used independently or as one or several modules in a college or high school class.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Ana Martins <strong>*</strong>, <strong>Paola Vera-Licona</strong> <strong>*</strong>, Reinhard Laubenbacher
    
  </div>

  <div class="pub-publication">
    
    Ledder G, Carpenter JP, Comar TD, editors. Washington: The Mathematical Association of America. Computational Systems Biology: Discrete Models of Gene Regulation Networks 189-200p,
    
    2013
  </div>

  <div class="pub-links">
    



<a class="btn btn-primary btn-outline btn-xs" href="https://pdfs.semanticscholar.org/804d/e53c822177fa61bca66f5a93fabd857d4ca0.pdf" target="_blank" rel="noopener">
  Preprint
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://www.maa.org/press/ebooks/undergraduate-mathematics-for-the-life-sciences" target="_blank" rel="noopener">
  PDF
</a>














<a class="btn btn-primary btn-outline btn-xs" href="http://admg.vbi.vt.edu/home/Outreach/Workshops/2" target="_blank" rel="noopener">
  Materials
</a>


  </div>

</div>

          
        </div>

        
      </div>

    </div>
  </div>
</div>
<footer class="site-footer">
  <div class="container">
    <p class="powered-by">

      &copy; 2018 &middot; 

      Powered by the
      <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> for
      <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a>.

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>


<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close btn-large" data-dismiss="modal">&times;</button>
        <h4 class="modal-title">Cite</h4>
      </div>
      <div>
        <pre><code class="modal-body tex"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-primary btn-outline js-copy-cite" href="#" target="_blank">
          <i class="fa fa-copy"></i> Copy
        </a>
        <a class="btn btn-primary btn-outline js-download-cite" href="#" target="_blank">
          <i class="fa fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

    

    
    

    

    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" integrity="sha512-3P8rXCuGJdNZOnUx/03c1jOTnMn3rP63nBip5gOP2qmUh5YAdVAvFZ1E+QLZZbC1rtMrQb+mah3AfYW11RUrWA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.3/imagesloaded.pkgd.min.js" integrity="sha512-umsR78NN0D23AzgoZ11K7raBD+R6hqKojyBZs1w8WvYlsI+QuKRGBx3LFCwhatzBunCjDuJpDHwxD13sLMbpRA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>
    
    
    <script src="/js/hugo-academic.js"></script>
    

    
    

    
    
    <script type="text/x-mathjax-config">
        MathJax.Hub.Config({ tex2jax: { inlineMath: [['$','$'], ['\\(','\\)']] } });
    </script>
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS_CHTML" integrity="sha512-tOav5w1OjvsSJzePRtt2uQPFwBoHt1VZcUq8l8nm5284LEKE9FSJBQryzMBzHxY5P0zRdNqEcpLIRVYFNgu1jw==" crossorigin="anonymous"></script>
    
    

  </body>
</html>

